<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsBottomUpPage</class>
 <widget class="QWidget" name="ResultsBottomUpPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>199</height>
   </rect>
  </property>
  <property name="toolTip">
   <string>Inspect the profile data samples in an aggregated view, showing the bottom-up call-graph tree.</string>
  </property>
  <layout class="QVBoxLayout" name="bottomUpVerticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="bottomUpSearch">
     <property name="toolTip">
      <string>Filter the call graph tree.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="bottomUpTreeView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="costAggregationLabel">
        <property name="text">
         <string>Aggregate cost by:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="costAggregationComboBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
